src/controllers/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './auth.controller';
 export * from './user.controller';
 export * from './role.controller';
+export * from './admin.disable.account.controller';
```

Comment:
Please delete this controller and put the the function inside it in the already existing appropriate controller.